profiles/definitions/latest-2019-04-30.md

```diff
@@ -41,6 +41,21 @@ profiles:
           - managedClusters/accessProfiles
         '2017-07-01':
           - containerServices
+      microsoft.blockchain:
```

Comment:
What's this update for ? could you add someone relevant to review ?

Reply:
I did not make this change; it was pulled in, because the master branch was renamed to main branch and this change was already there - made by Mark Cowlishaw via commit https://github.com/Azure/azure-rest-api-specs/commit/1bcc20d7c3523dfddc34a44881048824f3b86d1c.  Since my fork was off of master, when the rename happened, this got pulled in.  I tried redoing my changes; but could not get rid of this.  If you know how I can remove, please let me know.

Reply:
@markcowl , can you please help - either approve your changes that I pulled in my PR, or pl. advise how I can avoid including your changes?